url.bs

```diff
@@ -3411,42 +3410,42 @@ method steps are:
 <p>The <dfn method for=URLSearchParams><code>delete(<var>name</var>)</code></dfn> method steps are:
 
 <ol>
- <li><p>Remove all name-value pairs whose name is <var>name</var> from
- <a for=URLSearchParams>list</a>.
+ <li><p><a for=list>Remove</a> all <a for=/>tuples</a> whose name is <var>name</var> from
```

Comment:
"whose first element"? And similarly below.

Reply:
I was trying to strike a balance between going full `<dfn>` and being rather opaque.